src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Object> get() {
```

Comment:
The return type of your `get()` method should not be a List of Objects. It should be a List of Bulldozers. Please, change the return type to `List<Bulldozer>`.